react-components/src/components/RevealToolbar/ResetCameraButton.tsx

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement, useCallback } from 'react';
+
+import { Button, Tooltip as CogsTooltip } from '@cognite/cogs.js';
+import { useTranslation } from '../i18n/I18n';
+import { useCameraNavigation } from '../../hooks/useCameraNavigation';
+import { useSceneDefaultCamera } from '../../hooks/useSceneDefaultCamera';
+
+type ResetCameraButtonProps = {
+  sceneExternalId?: string;
+  sceneSpaceId?: string;
+};
+
+export const ResetCameraButton = ({
+  sceneExternalId,
+  sceneSpaceId
+}: ResetCameraButtonProps): ReactElement => {
+  const { t } = useTranslation();
+  const cameraNavigation = useCameraNavigation();
+
+  const resetCameraToHomePosition = useCallback(() => {
+    if (sceneExternalId !== undefined && sceneSpaceId !== undefined) {
+      useSceneDefaultCamera(sceneExternalId, sceneSpaceId);
```

Comment:
I think its forbidden by react to do something like this. You can't run hooks conditionally. Also the API for the hook is the same as for main camera navigation hook, i.e. you should destructure or directly call the function that hook returns, not just call the hook itself. Also, scene that you can test it with on 3D test is called `savelii_scene1`, it has some default camera position.